cou/steps/__init__.py

```diff
@@ -171,14 +171,23 @@ def canceled(self) -> bool:
 
     @property
     def results(self) -> Any:
-        """Return result of UpgradeStep."""
+        """Return result of BaseStep."""
         return self._task.result() if self._task is not None else None
 
-    def add_step(self, step: UpgradeStep) -> None:
+    @property
+    def is_functionless(self) -> bool:
+        """Check if this is a functionless step.
+
+        :return: True if doesn't contain function. False otherwise.
+        :rtype: bool
+        """
+        return self._coro is None
```

Comment:
I didn't understand why we are adding this property if we already implemented `__bool__` that checks self._coro on itself and on the sub_steps. Is it not enough use what we already have?